session/cookie.js

```diff
@@ -124,6 +124,8 @@ Cookie.prototype = {
     }
   },
 
+  set data(data) {},
```

Comment:
Based on the test provided, calling this should show a deprecation message to alert users this will be removed in the version 2.0

Reply:
I updated the PR to remove the setter.

Reply:
Seems good